packages/sdks/web-component/src/lib/descope-wc/BaseDescopeWc.ts

```diff
@@ -307,6 +307,14 @@ class BaseDescopeWc extends HTMLElement {
         try {
           const resp = await origFn(...args);
           return resp;
+        } catch (e) {
+          // return a generic error object in case of an error
+          return {
+            error: {
+              errorCode: 'E151000',
```

Comment:
Use a different error code, so we can tell it is coming from here

Reply:
Which one?

Reply:
good point Eliya, `E` is used by Descope service,

in go-sdk Descope uses `G...` for generated errors (not sure g stands for generated or go btw)

![image](https://github.com/user-attachments/assets/7c1e8c4f-cb3b-4b12-b39c-26d22ad8d954)


maybe `G` or dedicated prefix for this (`J` for JS?)

Reply:
This is what @dorsha suggested in the issue
https://github.com/descope/etc/issues/7166#issuecomment-2249727456

Reply:
We can use J

Reply:
So `J151000` in both places?

Reply:
`J151000` here and `J151001` in `packages/sdks/web-component/src/lib/descope-wc/DescopeWc.ts` if we don't have any error at all

Reply:
Done